### PR TITLE
ADD: Option to configure Prometheus Data Retention

### PR DESCRIPTION
**Pull request by PatrickRL**
ADD: option to configure data retention of Prometheus

### CODE REVIEW

File: launcher/src/store/services.js [hunk 1]
```diff
@@ -423,9 +423,17 @@ export const useServices = defineStore("services", {
           hIcon: "/img/icon/service-icons/prometheus.png",
           linkUrl: "",
           docsUrl: "https://prometheus.io/docs/introduction/overview/",
-          headerOption: true,
+          headerOption: false,
```
Comment (NeoPlays) [inline launcher/src/store/services.js:426]: Do you need to change this?

**Review by NeoPlays — CHANGES_REQUESTED** [review]

File: launcher/src/store/services.js (hunk 1 above)
Comment (NeoPlays) [reply launcher/src/store/services.js:426]: with this open are you removing the header icon:
grafik

Comment (PatrickRL) [reply launcher/src/store/services.js:426]: Accidentally left this change after testing something

**Review by NeoPlays — APPROVED** [review]
👍🏼

**Review by NeoPlays — APPROVED** [review]

**Pull request merged**